packages/headless-wallet-button-integration/src/headless-wallet-button-integration-request-service.ts

```diff
@@ -0,0 +1,117 @@
+import { RequestSender, Response } from '@bigcommerce/request-sender';
+
+import {
+    BillingAddress,
+    BillingAddressUpdateRequestBody,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { GraphQLRequestOptions } from './graphql-request-options';
+
+interface UpdateHeadlessBillingAddressResponse {
+    data: {
+        site: {
+            checkout: {
+                billingAddress: {
+                    entityId: string;
+                    address1: string;
+                    address2: string;
+                    city: string;
+                    company: string;
+                    countryCode: string;
+                    email: string;
+                    firstName: string;
+                    lastName: string;
+                    phone: string;
+                    postalCode: string;
+                    stateOrProvince: string;
+                    stateOrProvinceCode: string;
+                };
+            };
+        };
+    };
+}
+
+export default class HeadlessWalletButtonIntegrationRequestService {
```

Comment:
This class will be included all required graphql requests for wallet button initialization process, such as: `updateBillingAddress/addBillingAddress`, `updateShippingAddress/addSippingAddress`, `updateShippingOption, getShippingCoutryList`

Reply:
Yes, sure I am working on it. Thanks